codecov.yml

```diff
@@ -47,6 +47,6 @@ coverage:
         if_ci_failed: error
 
 comment:
-  layout: "header, diff, changes, sunburst, suggestions, tree"
+  layout: "tree"
   branches: null
-  behavior: once
+  behavior: default
```

Comment:
This behavior modification is meant to add a new comment to the PR when a new commit is added rather than updating the old comment.  This will be more useful in seeing the history - especially if the later commits are to fix the coverage.